### PR TITLE
hq task list, info

### PR DESCRIPTION
**Pull request by User3574**
Issue: #426, #458

This PR should serve as a base for additional (https://github.com/It4innovations/hyperqueue/issues/458) issues. Now it just solves a problem with a big task list output by moving some of the info into task info. Also the usage would be similar as is for a job (job list, job info).

### CODE REVIEW

File: crates/hyperqueue/src/client/task.rs [hunk 1]
```diff
@@ -32,7 +36,19 @@ pub struct TaskListOpts {
     pub verbosity: VerbosityFlag,
 }
 
-pub async fn output_job_tasks(
+#[derive(clap::Parser)]
+pub struct TaskInfoOpts {
+    /// Select specific job(s).
+    pub job_selector: JobSelectorArg,
```
Comment (Kobzol) [inline crates/hyperqueue/src/client/task.rs:42]: We assume a single job here, right? So this should not allow passing multiple job IDs.

**Review by Kobzol — COMMENTED** [review]
Could you please write a more detailed description on what this PR does/what it changes?

File: crates/hyperqueue/src/client/task.rs (hunk 1 above)
Comment (User3574) [reply crates/hyperqueue/src/client/task.rs:42]: Yes, you're right.
This PR should serve as a base for additional (#458) issues. Now it just solves a problem with a big task list output by moving some of the info into task info. Also the usage would be similar as is for a job (job list, job info).

**Comment by spirali** [comment]
@Kobzol The goal is to provide detailed information per task (in the same fashion list/info as for jobs).

The change is trying to solve #458.

@User3574 LGTM. In task info, I propose vertical oriented table (as for job info) as more individual information per task would be needed.

**Review by Kobzol — APPROVED** [review]

**Pull request merged**